src/common/iconbndl.cpp

```diff
@@ -192,6 +206,75 @@ void wxIconBundle::AddIcon(wxInputStream& stream, wxBitmapType type)
 
 #endif // wxUSE_STREAMS && wxUSE_IMAGE
 
+
+#ifdef __WINDOWS__
+
+namespace {
+
+// struct declarations taken from https://msdn.microsoft.com/en-us/library/ms997538.aspx
+#pragma pack(push)
```

Comment:
I know little about GCC but it seems that it does support this pragma: https://gcc.gnu.org/onlinedocs/gcc-4.0.4/gcc/Structure_002dPacking-Pragmas.html